### PR TITLE
Adding styles and reducing bundle size to frontend's ProgressBar component

### PR DESCRIPTION
**Pull request by bansalsurya**
Signed-off-by: Surya Bansal <2647surya@gmail.com>

## Proposed changes
Isuue #2100  : Change RC-Progress component to Material UI Compenent and decrease the bundle size. 
## Types of changes

-Added LinearProgress component from Material-UI.
-Added conditions to decide bar color.
-Added colors from the Material-UI theme. 
-Added custom styling of the bar.
-Dynamic response.
-All Unit tests(Specs) passed.

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- No extra dependancy, used @material-ui/core, which was already present.

## Special notes for your reviewer:
Thank you for reviewing my code! It was great to work on it and be a part of litmus community!

### CODE REVIEW

**Comment by rajdas98** [comment]
Thanks, @bansalsurya for recreating this PR. We will merge this PR asap.

File: litmus-portal/frontend/src/components/ProgressBar/LinearProgressBar/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/ProgressBar/LinearProgressBar/styles.tsx:1]: You may change the file extension to .ts as it’s not having some component.

File: litmus-portal/frontend/src/components/ProgressBar/AnalyticsLinearProgressBar/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/ProgressBar/AnalyticsLinearProgressBar/styles.tsx:1]: Please change the extension to .ts like we have for other style files @bansalsurya

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

**Review by ishangupta-ds — DISMISSED** [review]
LGTM

**Comment by ishangupta-ds** [comment]
@bansalsurya changes are good but please do an ‘npm run format’ in frontend directory, the frontend services checks are failing in CI, also do sign the commits for DCO check to pass.

**Comment by ishangupta-ds** [comment]
@rajdas98 this may be merged after changing the PR name, it can mention the component which is being changed. Cc: @bansalsurya

**Review by ishangupta-ds — APPROVED** [review]

**Review by rajdas98 — APPROVED** [review]

**Comment by rajdas98** [comment]
Thanks, @bansalsurya for this work 🙂. I am going to merge this PR. Hope to see more contribution from your side.

**Pull request merged**